source/user-manual/reference/tools/wazuh-keystore.rst

```diff
@@ -31,3 +37,10 @@ Example
 
     # /var/ossec/bin/wazuh-keystore -f indexer -k username -v admin
     # /var/ossec/bin/wazuh-keystore -f indexer -k password -v admin
+
+* Alternate ways to set the password:
+
+.. code-block:: console
+
+    # /var/ossec/bin/wazuh-keystore -f indexer -k username -vp /file/with/password
+    # echo "SecretPassword" | /var/ossec/bin/wazuh-keystore -f indexer -k password
```

Comment:
Maybe its better to set this as the default example (because is safer) and the alternative to be the original one

Reply:
You are right. Changed.